ci-operator/step-registry/sandboxed-containers-operator/install-trustee-operator/sandboxed-containers-operator-install-trustee-operator-commands.sh

```
@@ -0,0 +1,702 @@
+    oc logs pod/${kbs_client_pod} -n ${kbs_client_namespace} || true
+    oc delete pod/${kbs_client_pod} -n ${kbs_client_namespace} --ignore-not-found=true
+    return 1
+  fi
+
+  echo ">>> kbs-client pod is ready"
+
+  # Test basic connectivity to Trustee KBS
+  echo ">>> Testing connectivity to Trustee KBS at ${TRUSTEE_URL}"
+
+  # Try to fetch the auth endpoint (basic connectivity test)
+  if oc exec ${kbs_client_pod} -n ${kbs_client_namespace} -- \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)